### ISSUE TITLE
Add Tag Snapshot Block to Report Page

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#243]
## What problem is the feature trying to solve?
With issue #204, we've identified more useful reporting for accessibility pros. One of the key missing features is a tag snapshot that shows the number of issues related to critical tags. 

## Desired Solution
On a single report page (see #232), the tag snapshot would show tags and the number of alerts per tag. We would highlight tags with the highest weight and the most issues. For instance, if a "WCAG 2.2" tag had a weight of "2" and a "Best Practices" tag had a weight of "1", we would show "WCAG 2.2".

3-6 tags will be shown on the page, depending on what blocks are being displayed.

3 tags are shown if Grade + Timeline is present.  6 tags are shown if only Grade or Timeline is present. 

Users would be able to toggle this block on and off and set the weights of tags from the report editor page (see #230).

## Technical Details 
- We may need to consider how this feature queries content, as mentioned in #232. 
- We'll use basic styles from [Bootstrap](http://getbootstrap.com)

## Steps to Completion

- [ ] Develop a "dummy" Twig template, using sample data in the controller.
- [ ] Confirm features via User Testing.
- [ ] Make updates from User Testing lessons.
- [ ] Replace sample data with actual features.
- [ ] Integrate page and related features into the core codebase.

**Comment by bbertucc** [comment]
This was completed in the yet-to-be-released V1
